README.md

```diff
@@ -75,9 +75,9 @@ Then, acquire the source code by cloning the git repository:
 
 Be sure to also configure your system to use the appropriate proxy settings, e.g. by setting the `https_proxy` and `http_proxy` variables.)
 
-By default you will be building the latest unstable version of Julia. However, most users should use the most recent stable version of Julia, which is currently the `0.5` series of releases. You can get this version by changing to the Julia directory and running
+By default you will be building the latest unstable version of Julia. However, most users should use the most recent stable version of Julia, which is currently the `0.6` series of releases. You can get this version by changing to the Julia directory and running
 
-    git checkout release-0.5
+    git checkout release-0.6
```

Comment:
Should this point to the tag instead? `git checkout v0.6.0` or whatever, since the head of the `release-0.6` branch is not the same as the latest release?